plugins/modules/change_request.py

```diff
@@ -352,7 +352,7 @@ def build_payload(module, table_client):
 
     if module.params["template"]:
         standard_change_template = table.find_standard_change_template(
-            table_client, module.params["template"]
+            table_client, module.params["template"], module.params["template_table"]
```

Comment:
Could you please add documentation and tests for this parameter? 

Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.